addons/actions/src/lib/types/class/createFakeConstructor.js

```diff
@@ -0,0 +1,12 @@
+import createFunction from '../function/createFunction';
+
+export default function createFakeConstructor(obj, key) {
+  const Func = createFunction(obj[key]);
+  const func = new Func();
```

Comment:
I'm a bit confused by the naming here. The variable is called `func`, but it's not a function. The method is called `createFakeConstructor`, but it returns an instanciated object not the constructor.

And probably the whole `classType` should be renamed to `objectType`, because it actually applies to any object that doesn't pass other type checks.

Reply:
I've renamed the variables and functions and moved them to`./types/object` instead of `./types/class`. See: 5b9e213aff19678b377130ca828bfb2cf05608bd